support-frontend/app/controllers/CreateSubscriptionController.scala

```diff
@@ -210,6 +210,12 @@ class CreateSubscriptionController(
           s"gu.contributions.recurring.contrib-timestamp.$billingPeriod" -> DateTime.now.getMillis.toString,
           "gu_recurring_contributor" -> true.toString,
         )
+      case _: SupporterPlus =>
+        List(
+          "gu_digital_subscriber" -> true.toString, // TODO: confirm these
```

Comment:
good question - is editions access triggered off this field?

Reply:
that would be a more important question on the members-data-api side.
Also, this would mean even people contributing below the thresold would get ad free and etc for a day?

Reply:
Anyone on supporter plus is paying over the threshold